lib/plugins/inspec-suggest/test/functional/suggtool_bump_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative "../helper"
+require "tmpdir" unless defined?(Dir.mktmpdir)
+
+describe "inspec suggtool bump" do
+  include CorePluginFunctionalHelper
+
+  let(:fixture_path) { File.join(repo_path, "lib", "plugins", "inspec-suggest", "test", "fixtures") }
+  let(:config) { File.join(fixture_path, "sugg-conf", "suggest-bump.yaml") }
+
+  describe "when packaging the basic test fixture set" do
+    it "bump:patch version default" do
```

Comment:
Need to have a separate test for explicitly setting patch.

Reply:
Added separated test explicitly for patch